src/main/java/org/folio/innreach/mapper/MappingMethodsToEntity.java

```diff
@@ -0,0 +1,20 @@
+package org.folio.innreach.mapper;
+
+import org.springframework.stereotype.Component;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Date;
+import java.util.UUID;
+
+@Component
+public class MappingMethodsToEntity {
```

Comment:
this methods can be added to existing MappingMethods class if they are really needed

Reply:
removed